cmd/swapd/profile.go

```diff
@@ -21,8 +21,14 @@ func maybeStartProfiler(c *cli.Context) error {
 	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
+	server := &http.Server{
+		Addr:              bindIPAndPort,
+		ReadHeaderTimeout: time.Second,
```

Comment:
This is just the profiler, but I decided to add the timeout here instead of disabling the lint complaint.